### PR TITLE
Added useModal Prop

### PR DESCRIPTION
**Pull request by narteysarso**
useModal prop allows you to specify whether the panel is implemented as a <Modal />  or as a <View /> component as certain libraries do not interact smoothly with Modals

### CODE REVIEW

File: src/BottomSheet/index.js [hunk 1]
```diff
@@ -82,56 +104,90 @@ class BottomSheet extends Component {
...
+        </Modal>
+          : modalVisible && this.setModalVisible &&
+          <View
+            style={[
+              styles.wrapper,
+              {
+                backgroundColor: backgroundColor || "#25252599",
+                height: height,
+                backgroundColor: backgroundColor || 'transparent',
+                position: 'absolute',
```
Comment (victorcosta) [inline src/BottomSheet/index.js:180]: Do you think we can change these inline styles for the style file??

**Comment by kcotias** [comment]
Hey bro! @narteysarso First of all, thanks a lot for your contribution! Could you explain a little bit about your PR? Just a short description of what it adds!

File: src/BottomSheet/index.js (hunk 1 above)
Comment (narteysarso) [reply src/BottomSheet/index.js:180]: okay, I will look into it

**Comment by kcotias** [comment]
Closing due to inactivity!

**Pull request closed without merging**